<ui version="4.0" >
 <class>InstallFrm</class>
 <widget class="QDialog" name="InstallFrm" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Install Rockbox</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="8" row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="rbutilqt.qrc" >:/icons/icons/wizard.xpm</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Select your device in the filesystem</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="lineMountPoint" />
   </item>
   <item row="1" column="3" >
    <widget class="QToolButton" name="buttonBrowse" >
     <property name="text" >
      <string>&amp;Browse</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Please select the Rockbox version you want to install on your player:</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Version</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QRadioButton" name="radioStable" >
        <property name="text" >
         <string>Rockbox &amp;stable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioArchived" >
        <property name="text" >
         <string>&amp;Archived Build</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioCurrent" >
        <property name="text" >
         <string>&amp;Current Build</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="3" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Details</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="labelDetails" >
        <property name="text" >
         <string>Details about the selected version</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="labelNote" >
        <property name="text" >
         <string>Note</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" >
    <widget class="QCheckBox" name="checkBoxCache" >
     <property name="toolTip" >
      <string>Rockbox Utility stores copies of Rockbox it has downloaded on the local hard disk to save network traffic. If your local copy is no longer working, tick this box to download a fresh copy.</string>
     </property>
     <property name="text" >
      <string>&amp;Don't use locally cached copy</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>185</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="2" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rbutilqt.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InstallFrm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InstallFrm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
